http-server/server/server.go

```diff
@@ -11,17 +12,31 @@ import (
 )
 
 var pageAccessCount = 0
-var Param string
+var param string = ""
 
 var greetings = []string{"Hello", "Greetings", "Welcome", "Hi"}
 var titles = []string{"the Mighty Traveler", "the Great Summoner", "the Conqueror of Titans", "the Destroyer of Worlds", "the King of Oceans", "the King of Underworld"}
 
+// GetParam returns pointer to param variable (for other packages)
+func GetParam() *string {
```

Comment:
Would be better to return plain string, not pointer, see: https://dhdersch.github.io/golang/2016/01/23/golang-when-to-use-string-pointers.html

Reply:
rewrote using plain strings